benchmarks/Exporter/JaegerExporterBenchmarks.cs

```diff
@@ -148,63 +147,73 @@ protected override void Dispose(bool disposing)
             }
         }
 
-        private SpanData CreateTestSpan()
+        private Activity CreateTestActivity()
         {
-            var startTimestamp = new DateTimeOffset(2019, 1, 1, 0, 0, 0, TimeSpan.Zero);
+            var startTimestamp = DateTime.UtcNow;
             var endTimestamp = startTimestamp.AddSeconds(60);
-            var eventTimestamp = new DateTimeOffset(2019, 1, 1, 0, 0, 0, TimeSpan.Zero);
-
+            var eventTimestamp = DateTime.UtcNow;
             var traceId = ActivityTraceId.CreateFromString("e8ea7e9ac72de94e91fabc613f9686b2".AsSpan());
-            var spanId = ActivitySpanId.CreateFromString("6a69db47429ea340".AsSpan());
+
             var parentSpanId = ActivitySpanId.CreateFromBytes(new byte[] { 12, 23, 34, 45, 56, 67, 78, 89 });
+
             var attributes = new Dictionary<string, object>
             {
-                { "stringKey", "value"},
-                { "longKey", 1L},
+                { "stringKey", "value" },
```

Comment:
Minor: I remember I've learned from @rajkumar-rangaraj that the new syntax allows us to be more expressive, see if we should encourage this style across the codebase:

```csharp
new Dictionary<string, string> {
    ["foo"] = "1",
    ["bar"] = "2",
}
```